firestore/src/include/firebase/firestore/aggregate_query_snapshot.h

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIREBASE_FIRESTORE_SRC_INCLUDE_FIREBASE_FIRESTORE_AGGREGATE_QUERY_SNAPSHOT_H_
+#define FIREBASE_FIRESTORE_SRC_INCLUDE_FIREBASE_FIRESTORE_AGGREGATE_QUERY_SNAPSHOT_H_
+
+#include <cstddef>
+#include <cstdint>
+
+namespace firebase {
+namespace firestore {
+
+class AggregateQuery;
+class AggregateQuerySnapshotInternal;
+
+/**
+ * @brief A QuerySnapshot contains zero or more DocumentSnapshot objects.
+ *
+ * QuerySnapshot can be iterated using a range-based for loop, and its size can
+ * be inspected with empty() and size().
+ *
+ * @note Firestore classes are not meant to be subclassed except for use in test
+ * mocks. Subclassing is not supported in production code and new SDK releases
+ * may break code that does so.
+ */
+class AggregateQuerySnapshot {
```

Comment:
This class-level comment appears to have been copied from the QuerySnapshot class. Can you fix it to be appropriate for the AggregateQuerySnapshot class?